pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -294,24 +294,38 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     }
   }
 
-  protected async syncDockerCompose() {
-    await Promise.all([
-      this.syncHostDockerCompose(),
-      ...(await this.supportedDistros).map(distro => this.syncDistroDockerCompose(distro.name)),
-    ]);
+  protected async syncDockerPlugins() {
+    const pluginNames = await this.getDockerCliPluginNames();
+
+    await Promise.all(pluginNames.map(async(pluginName) => {
+      await Promise.all([
+        this.syncHostDockerPlugin(pluginName),
```

Comment:
The plugin name is calculated from Linux, but we're syncing the host one instead. We should just make it a separate loop instead, in case there are any differences (maybe some plugin in the future wouldn't work in a WSL distro or something).